backend/script/test

```diff
@@ -8,5 +8,5 @@ poetry run black --check app
 poetry run mypy app
 poetry run ruff check app
 
-mkdir -p test-results
-poetry run pytest --junitxml=test-results/junit.xml
\ No newline at end of file
+mkdir -p ../test-results
```

Comment:
Use some shell magic to make this work from any cwd.

I use variations of 

```
cd "$(dirname "$0")/.."
```

to cd to the parent directory of the directory where the script is (the dirname $0 bit gives you the latter).